.github/workflows/main.yml

```diff
@@ -195,7 +197,7 @@ jobs:
           cygpath -aw / &&
           git.exe version --build-options >version &&
           cat version &&
-          grep "$(sed -e 's|^v||' -e 's|-|.|g' <bundle-artifacts/next_version)" version &&
+          grep "${{ steps.build-artifact.outputs.git-version }}" version &&
```

Comment:
on line 198 we have `--build-options` and up where the `...output.git-version` is set we do not use that option.  granted this does not matter since the grep will match.

should we consider using `--build-options` in both places and then verify both the version string *and* the `build from commit` lines ??

While at it, we could also verify that the `cpu` line matches the expected value from the matrix.  (This might be overkill.)

Reply:
@jeffhostetler Right. We could change how it was done previously (which _is_ where this paradigm is coming from, grepping for the version instead of requiring an exact match of the output with an expected result).

However, there are two reasons why I don't want to do that here:

- The purpose of this PR is to fix the brokenness in a minimal fashion. Before, we grepped for the empty string, which was incorrect. Now we grep for the version, which is correct.
- This code is actually not really stand-alone. It was copy/edited [from `git-sdk-32`](https://github.com/git-for-windows/git-sdk-32/blob/d818936674d95a984af49beaec8fb55a701cd068/.github/workflows/git-artifacts.yml#L219-L231) and that is where I need to make follow-up changes because I want to keep both workflows somewhat in sync.

Reply:
Oh, and the `cat version` is trying to help debugging, that's also the reason why `--build-options` is used there.